Formula/buildifier.rb

```diff
@@ -13,11 +13,96 @@ class Buildifier < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "5b54427b4bd78b0b3dccfd66b8003a52aab3b4d1a7683586b66d6a6c835c0b4b"
   end
 
-  depends_on "bazelisk" => :build
+  depends_on "go" => :build
 
   def install
-    system "bazelisk", "build", "--config=release", "buildifier:buildifier"
-    bin.install "bazel-bin/buildifier/darwin_amd64_stripped/buildifier"
+    ENV["GOPATH"] = buildpath
```

Comment:
This shouldn't be needed for modern projects.

Reply:
fixed in https://github.com/Homebrew/homebrew-core/pull/70921/commits/dce2d5b0c9ff89417ee17c362162979c363948f1